front-end/src/components/Login/Login.js

```diff
@@ -0,0 +1,18 @@
+import React, {useState} from 'react';
+import { useDispatch } from 'react-redux';
+import { setUser } from '../../reducers/user.js';
+
+// eslint-disable-next-line func-style
```

Comment:
We can adjust the linter as well so we do not have to use the disabling comment👍